client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/index.tsx

```diff
@@ -202,6 +208,64 @@ const SiteMigrationInstructions: Step< {
 		</div>
 	);
 
+	if ( useContainerV2 ) {
+		const title = translate( 'Let’s migrate your site' );
+		const topBar = (
+			<Step.TopBar rightElement={ <SupportNudge onAskForHelp={ navigateToDoItForMe } /> } />
+		);
+		const progressCircle = (
```

Comment:
We could also reuse the same component [here](https://github.com/Automattic/wp-calypso/blob/de6a282e8906f41f87c5186b357b8a6095d8a19f/client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/index.tsx#L185)